Source/UnitTests/Core/PowerPC/JitArm64/MovI2R.cpp

```diff
@@ -0,0 +1,128 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <cstddef>
+#include <type_traits>
+
+#include "Common/Arm64Emitter.h"
+#include "Common/Assert.h"
+#include "Common/BitUtils.h"
+#include "Common/Random.h"
+
+#include <gtest/gtest.h>
+#include <mbedtls/hmac_drbg.h>
+
+namespace
+{
+class PRNG
```

Comment:
Could we put this in Common/Random.h? I guess it'd have to be separate from the existing class as that one uses mbedtls_entropy_context to seed. The pimpl idiom should probably be used to avoid exposing mbedtls details directly.

Reply:
Done.